release/BUILD

```diff
@@ -182,6 +186,115 @@ py_test(
     ],
 )
 
+
+####
+# AIR smoke tests
+####
+
+
+py_test(
+    name = "air_benchmark_xgboost_smoke_test",
+    size = "medium",
+    srcs = test_srcs,
+    main = "air_tests/air_benchmarks/workloads/xgboost_benchmark.py",
+    tags = [
+        "exclusive",
+        "team:ml",
+    ],
+    deps = [
+        "//:ray_lib",
+        "//python/ray/air:ml_lib",
+    ],
+    args = ["--smoke-test"]
+)
+
+py_test(
+    name = "air_benchmark_data_smoke_test",
+    size = "small",
+    srcs = test_srcs,
+    main = "air_tests/air_benchmarks/workloads/data_benchmark.py",
+    tags = [
+        "exclusive",
+        "team:ml",
+    ],
+    deps = [
+        "//:ray_lib",
+        "//python/ray/air:ml_lib",
+    ],
+    args = ["--dataset-size-gb=1", "--num-workers=1"]
+)
+
+# py_test(
+#     name = "air_benchmark_gpu_batch_prediction_smoke_test",
```

Comment:
Should this one be commented out?

Reply:
Yes, this is blocked by https://github.com/ray-project/ray/issues/29524

Reply:
Oh can you add a comment for that too? (At a first glance I couldn't tell why this one was commented out, and why the training one was marked as missing with the related issue)